gigadb/app/tools/files-metadata-console/controllers/UpdateController.php

```diff
@@ -50,36 +48,42 @@ final class UpdateController extends Controller
     public string $stop = '0';
 
     /**
-     * Console command for updating size for all files in a given dataset
+     * Console command for updating md5 values for all files in a given dataset
      *
-     * ./yii update/file-sizes --doi=100142
+     * ./yii update/file-sizes --doi=100039
```

Comment:
**typo:** it should be `./yii update/md5-values --doi=100039

Reply:
Now updated.